cypress/integration/testBorna/test.cy.ts

```diff
@@ -0,0 +1,122 @@
+function makeid(length) {
```

Comment:
I wanted to use a random string that only contained lowercase letters, which I wasn't really sure how to achieve with uniqid. Also I didn't want numbers because the field in question which uses it accepts only lowercase letters.